### PR TITLE
ICT-108-media-kronos-passport

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

File: app/data/kronos/authorities.js [hunk 1]
```diff
@@ -0,0 +1,2109 @@
...
+        ],
+        "isParty": false,
+        "shortName": "Ethiopia"
+    },
+    {
+        "code": "EUR",
+        "name": "European Union",
+        "groups": [
+            "nationality",
+            "authority"
```
Comment (sbilodeau) [inline app/data/kronos/authorities.js:608]:
```suggestion
        "code": "EUE",
        "name": "European Union",
        "groups": [
            "authority"
```

File: app/data/kronos/nationalities.js [hunk 2]
```diff
@@ -0,0 +1,2019 @@
...
+    {
+        "code": "EUR",
+        "name": "European Union",
+        "groups": [
+            "nationality",
+            "authority"
+        ],
+        "isParty": false,
+        "shortName": "European Union"
+    },
```
Comment (sbilodeau) [inline app/data/kronos/nationalities.js:612]:
```suggestion
```

File: app/directives/kronos/passport.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+      scope: {
+        participant : "=?"       ,
+        request     : "=?"       ,
+        binding     : "=ngModel" ,
+        show        : '=?'       ,     
+      },
+      link:function($scope){
+        $scope.binding ={};
+
+        $http.get('/api/v2015/countries',{ params: {f:{code3:1,code:1} } }) .then(({data})=> $scope.countries = data) .then(loadBinding);
```
Comment (sbilodeau) [inline app/directives/kronos/passport.js:28]: should nationalities & authorities as Code3 have slight difference
see: 
https://www.icao.int/publications/Documents/9303_p3_cons_en.pdf#page=29

File: app/directives/kronos/passport.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+          else
+            $scope.binding = { ...$scope.binding, number,nationality, issuingAuthority, conferenceId, contactId, firstName, lastName, birthDate: dateOfBirth }
+        }
+
+        function getCode3(c){
+          const country = $scope.countries?.find(({code})=> code === c);
+
+          return country?.code3
+
+        }
```
Comment (sbilodeau) [inline app/directives/kronos/passport.js:135]: should gaia Countries passport nationalities

**Review by sbilodeau — CHANGES_REQUESTED** [review]

**Review by rqk2k — APPROVED** [review]

**Pull request merged**